src/main.cpp

```diff
@@ -332,7 +332,7 @@ void generate_supervised_data(const std::string& filename) {
       myprintf_so("Invalid game in %s\n", filename.c_str());
       break;
     }
-    myprintf_so("\rProcessed %d games", ++games);
+    myprintf_so("\rProcessed %d games\n", ++games);
```

Comment:
You can't do a \n at the end of a \r line -- it will flood the console :).

Reply:
Fixed!